src/App/components/FormComponents/Checkbox.js

```diff
@@ -41,6 +43,13 @@ const Checkbox = ({ id, checked, name, disabled, groupTitle, label, errorMessage
                         <label htmlFor={id} className="d-block">{label}</label>{"\n"}
                     </div>
                     {errorMessage ? <div className="help-block">{errorMessage}</div> : null}
+                    {hintText && <div className="hint-text">{hintText}</div>}
+                    {expandingHintText && 
+                    <label className="help-block-expander">
+                        <input disabled={disabled} type="checkbox"/>
+                        <span className="header"><span className="material-icons arrow">keyboard_arrow_down</span>{expandingHintText}</span>
+                        <div className="content">This information is less important and only a minority of users will need it or the text is very long. In this case; both.</div>
+                    </label>}
                 </>
```

Comment:
When disable state is active, disable hint text and change icon color for hint expander aswell.

![image](https://user-images.githubusercontent.com/44967773/164173549-94ed9926-bd88-400e-9fdf-38aacc0a2508.png)